build/Helix/RunTestsInHelix.proj

```diff
@@ -21,11 +21,12 @@
     <HelixCorrelationPayload Include="$(TestPayloadDir)" />
   </ItemGroup>
   
-  <!-- These files are generated by GenerateHelixWorkItems.ps1 as part of the build pipeline. They are not checked-in to the repo. -->
+  <!-- Note: These *.proj files are generated by GenerateHelixWorkItems.ps1 as part of the build pipeline. They are not checked-in to the repo. -->
   <Import Project="GeneratedWorkItems\MrtCore-MrmUnitTests.proj"/>
   <Import Project="GeneratedWorkItems\MrtCore-MrmBaseUnitTests.proj"/>
   <Import Project="GeneratedWorkItems\MrtCore-MrtCoreUnpackagedTests.proj"/>
 
-  <!-- Note: It is also possible to directly create work-items for Helix here instead of using the scripts. -->
+  <!-- Additional tests -->
+  <Import Project="$(ProjFilesPath)\DynamicDependencies-Tests.proj"/>
```

Comment:
>ProjFilesPath [](http://example.com/codeflow?start=21&length=13)

Just wondering, why `ProjFilesPath` instead of `GeneratedWorkItems` ?

Reply:
I cloned these steps from the IntegrationTests build pipeline. Trying to understand how they're alike but not has been, er, fun. Yeah. That's the word...

You're more the expert here than I. What's the right answer here?

Reply:
Yeah... "fun"....

For the internal repo, it looks like this get's passed to Helix via `/p:ProjFilesPath=$(helixWorkItemsDir)`. But that doesn't exist in the Foundation repo.

So I think for this we just want to use `GeneratedWorkItems` for now.